.evergreen/config.in.yml

```diff
@@ -142,7 +142,7 @@ functions:
         working_dir: "src"
         timeout_secs: 300
         binary: bash
-        args: 
+        args:
```

Comment:
I'm not familiar with the rest of the work in this project.  Do we have a ticket filed to document how to use the new test runner?  Or should we do that now?

Reply:
I consider this a bit of WIP, the top level main can still have usability improvements added to it as part of this ticket, this subtask gets Warren and I unblocked to migrate all the benchmarks over to the format that allows us to fork per benchmark.

I am of course happy to document it, but at this time it is just `npm start` in the subdirectory we implemented the new runner. I think we'll want more than that for how to set the existing env vars (same as before but not documented) and how to write new benchmarks (also not documented before).

Reply:
https://github.com/mongodb/node-mongodb-native/pull/4399/files#diff-b3c66a0a9165da881a61d1ee36a81a6f9f9044fc5b0bf7e6afb817c072d4642d

already kicked off a bit of docs on my next PR

Reply:
Okay, great.  That works for me